_config/config.yml

```diff
@@ -0,0 +1,11 @@
+---
+Name: tinymce-config
+---
+SilverStripe\Forms\HTMLEditor\HTMLEditorConfig:
+  default_config_definitions:
+    cms:
+      configClass: 'SilverStripe\TinyMCE\TinyMCEConfig'
```

Comment:
This sets the default editor for the `cms` config specifically - but not the default for new ones. This allows devs to swap out that editor for their own one or for our replacement in the future if they want to.